### ISSUE TITLE
Another column in xbps-query result?

### ISSUE THREAD

**Issue by aarnt** [void-linux/xbps#241]
First, thanks for your nice distro and pkg manager!

Right now I use "xbps-query -Rs -" to get a list of all available packages for OctoXBPS.
Could it be possible to add "Download size" column in the result list?

Thanks and keep up the good work!

**Comment by xtraeme** [comment]
Hi @aarnt !

could you please give me an example of what exactly do you need?

**Comment by aarnt** [comment]
Hi @xtraeme ! Of course.

OctoXBPS uses "xbps-query -Rs -" to get the list of all official Void packages (installed + non installed) and it just works.

It would be nice if we could get another column in that list showing the "filename-size" of each package. It can be the exact type of value "xbps-query -R pkgname" gives. 
Something like this:

```
[-] pkgname1-version         Description of pkgname1        filename-size of pkgname1
[*] pkgname2-version         Description of pkgname2        filename-size of pkgname2
```

Instead of the current list:

```
[-] pkgname1-version         Description of pkgname1
[*] pkgname2-version         Description of pkgname2
```

Maybe it could be a xbps-query argument to add another column. I don't know what you all think it's best.

Thank you.

**Comment by xtraeme** [comment]
I see, I will come up with something in some hours. Cheers

**Comment by aarnt** [comment]
Ok, thank you.
There is a guy that made some adaptations in xbps-query code 2 years ago:

https://github.com/veksha/octoxbps-mod/tree/master/xbps-query-custom

I don't know if that helps, but...

**Comment by Duncaen** [comment]
imho this is best implemented by adding a format option, that allows to reference package dictionary keys.
```
$ xbps-query -f "%{state} %{filename-size} %{pkgver} \"%{short_desc}\"" -s ''
installed 908B base-system-0.113_2 "Void Linux base system meta package"
```

**Comment by xtraeme** [comment]
I agree, maybe we should add a format option to print any obj in the dictionary...

**Comment by aarnt** [comment]
It would be really nice.
